app/views/main.scala.html

```diff
@@ -73,6 +73,21 @@
         </main>
     </div>
 
+    @if(isLoggedIn) {
+    <dialog class="timeout-dialog">
```

Comment:
At the moment on the `/upload` page the token is continuously updated, for 12 hours, to ensure a user does not get logged out whilst their files are being uploaded. There is a question as to whether:

1. this behaviour should be retained; or
2. it should be removed, as they would then get this session timeout warning.

I think this question should be asked to the wider team, including Kirsten and Pauline.

Reply:
As discussed today. We will be retaining the behavior of the upload page.